include/vhost_v1.hrl

```diff
@@ -0,0 +1,5 @@
+-define(is_vhost_v1(V), is_record(V, vhost, 3)).
+
+-define(vhost_v1_field_name(V), element(2, V)).
+-define(vhost_v1_field_limits(V), element(2, V)).
+-define(vhost_v1_field_metadata(V), #{}).
```

Comment:
This header represents exactly the record fields. `metadata` isn't part of that: compilation should fails if we try to access this field in an older vhost record.